verification/example_config.toml

```diff
@@ -9,3 +9,7 @@ request_timeout = 10
 [solidity]
 compilers_list_url = "https://raw.githubusercontent.com/blockscout/solc-bin/main/list.json"
 refresh_versions_schedule = "0 0 * * * * *"
+
+[metrics]
```

Comment:
May be add "enabled" option, so people could disable metrics altogether (or even disable them by default)

Reply:
I agree that it would be better to disable them by default

Reply:
I think it should be enabled by default, as we did with solidity and sourcify 
<img width="926" alt="image" src="https://user-images.githubusercontent.com/41516657/181725352-2973693b-feaf-4e68-9607-dc7182b89642.png">

Reply:
It is not the main logic of the service and I believe users will not want to have it opened by default. So, I believe auto-enabling of `solidity` and `sourcify` should not be the reason to enable metrics by default

Reply:
ok, changed to `enabled: false` in default